api/openapi.yaml

```diff
@@ -500,3 +519,29 @@ components:
           example: "this is an error message"
       required:
         - message
+
+    Timestamp:
+      type: string
+      description: Date and time (RFC 3339) representing the UTC timestamp of the health check
+      example: 2021-10-23T15:05:37Z
```

Comment:
missing quotes around example string

Reply:
Is it because it is a string or for consistency? because it is not required for valid YAML

Reply:
removed timestamp so no longer required for 0.6.0